scripts/compile.sh

```diff
@@ -10,7 +10,7 @@ fi
 PROVIDER_NAME="docker"
 TARGET_DIR="$(pwd)/results"
 XC_ARCH=${XC_ARCH:-"386 amd64 arm"}
-XC_OS=${XC_OS:=linux darwin windows freebsd openbsd solaris}
+XC_OS=${XC_OS:=linux darwin windows openbsd solaris}
```

Comment:
What was the reason the remove `freebsd` here?

Reply:
One of the "archive" depending packages is not compiling on FreeBSD with go >= 1.12. 

https://github.com/moby/moby/pull/38818

Wanted to verify before proceeding, is it acceptable to remove freebsd or should I try to find alternative.
There is a benefit in using docker/cli packages in having exactly the same way of behaviour as docker-cli ensuring compatibility (e.g. support .dockerignore).

Reply:
One of the "archive" depending packages is not compiling on FreeBSD with go >= 1.12. 

https://github.com/moby/moby/pull/38818

Wanted to verify before proceeding, is it acceptable to remove freebsd or should I try to find alternative.
There is a benefit in using docker/cli packages in having exactly the same way of behaviour as docker-cli ensuring compatibility (e.g. support .dockerignore).

Reply:
@mavogel what would you like me to do with this one?

Reply:
I'll reach out to the hashicorp folks and ask them. I'll come to to you here. 
IMHO it makes total sense to use `docker/cli` packages for reusing functionality

Reply:
@akomic sorry for the delay. As part of the terraform `0.13` migration this provider will be forked off to a _community_ maintained one, meaning we will have full control over how to build and release it to the terraform registry via GH actions. 
TL;DR: I'll merge all remaining PRs, including this one, and then fork off. IMHO we'd go with `docker/cli`

Reply:
@mavogel Great, there are some things I'd like to add/improve regarding this functionality but wanted to see if it is not futile. I'll continue with contribution once it's forked.

Thanks

Reply:
@mavogel Great, there are some things I'd like to add/improve regarding this functionality but wanted to see if it is not futile. I'll continue with contribution once it's forked.

Thanks